content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -18,14 +18,7 @@ For information how to create a cluster with kubeadm once you have performed thi
 ## {{% heading "prerequisites" %}}
 
 
-* One or more machines running one of:
-  - Ubuntu 16.04+
-  - Debian 9+
-  - CentOS 7+
-  - Red Hat Enterprise Linux (RHEL) 7+
-  - Fedora 25+
-  - HypriotOS v1.0.1+
-  - Flatcar Container Linux (tested with 2512.3.0)
+* A compatible Linux host. The Kubernetes project does not maintain a matrix of supported Linux distributions, but we do have generic instructions for distributions based on Debian and Red Hat, and those without a package manager.
```

Comment:
Hi @chrisnegus .
nit: Could you remove "we". Possibly reword, something like:
A compatible Linux host. The Kubernetes project provides generic instructions for Linux distributions based on Debian and Red Hat, and those distributions without a package manager.
Thanks!

Reply:
Sure. I like that wording. Done. Let me know if you think I need to do anything else before the PR is merged.